options/gitignore/Bazel

```diff
@@ -6,7 +6,7 @@
 /bazel-*
 
 # Directories for the Bazel IntelliJ plugin containing the generated
-# IntelliJ project files and plugin configuration. Seperate directories are
+# IntelliJ project files and plugin configuration. Separate directories are
```

Comment:
I think this may technically get overridden if we ever regen these gitignores, but lgtm for the other changes